core/src/main/kotlin/com/alecstrong/sql/psi/core/SqlCoreEnvironment.kt

```diff
@@ -37,6 +39,11 @@ private object ApplicationEnvironment {
 
   val coreApplicationEnvironment: CoreApplicationEnvironment by lazy(mode = LazyThreadSafetyMode.SYNCHRONIZED) {
     CoreApplicationEnvironment(Disposer.newDisposable()).apply {
+      Logger.setFactory { object : DefaultLogger(it) {
```

Comment:
Why do you have to pass `it`? Would be nice if this no-op logger was a stateless singleton.

Reply:
It’s just some string, I think I could just pass an empty string and it would still work so I’ll do that